docs/guides/use_checkpointing.md

```diff
@@ -70,11 +88,13 @@ variables = model.init(key2, x1)
 
 # Flax's TrainState is a pytree dataclass and is supported in checkpointing.
 # Define your class with `@flax.struct.dataclass` decorator to make it compatible.
-tx = optax.sgd(learning_rate=0.1)      # An Optax SGD optimizer.
+tx = optax.sgd(learning_rate=0.001)      # An Optax SGD optimizer.
 state = train_state.TrainState.create(
     apply_fn=model.apply,
     params=variables['params'],
     tx=tx)
+# Make some simple fake grad update like the ones done in normal training.
```

Comment:
Suggestion:

Since it's only a forward pass and the example in the guide is just a toy example, then maybe:

```suggestion
# Perform a simple gradient update similar to the one during a normal training workflow.
```

(Internally, `.apply_gradients` Flax and Optax (`tx`) code, isn't it?)